README.rst

```diff
@@ -211,7 +212,7 @@ If you choose to opt for automated method you must first SSH into your Ansible c
 
 .. code-block:: console
 
-   ssh ${ssh_user}@${ansible_ip}
+   ssh cloud-user@$(terraform output -raw ansible_control_access_ip_v4)
```

Comment:
Why remove `${ssh_user}`?

Reply:
I think this was a discrepancy between my local branch when starting work on this. Also it is the same as the `ansible_user`` from above.

Reply:
Yeah @markgoddard @jackhodgkiss we should deffo leave it as what it was previously.